docs/arcadia-users-group/20221017-conda/lesson.md

```diff
@@ -0,0 +1,369 @@
+</figure>
+</center>
+
+For this lesson, we'll install miniconda. 
+We've included the latest installation instructions below.
+You can access the latest and legacy installation links [here](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links).
+
+#### Installing conda on an Apple machine
+
+Apple now has two processor types, the Intel x64 and the Apple M1 (or M2) ARM64.
+As of October 2022, many of the packages available via conda-forge and other conda installation channels are only available of the Intel x64 processor. 
+Therefore, even if you have an M1 (or M2) Mac, we currently recommend that you use the Intel x64 installation and take advantage of Apple's translation program Rosetta. 
+This requires a little bit of pre-configuration to make sure your Terminal application also runs with Rosetta.
+To set this up, open **Finder** -> navigate to your **Applications** folder -> right click on your Terminal application (either Terminal or iTerm2) -> select **Get Info** -> check the box for **Open using Rosetta**.
+
+<center>
+<figure markdown>
+  ![installingconda](rosetta.png){ width="200" }
+  <figcaption> Running Terminal/iTerm using Rosetta </figcaption>
+</figure>
+</center>
+
+After that is configured, open your Terminal application and copy and paste the following commands to download and install miniconda.
+
+```
+curl -JLO https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+bash Miniconda3-latest-MacOSX-x64_64.sh 
+```
+
+This will prompt you to accept the miniconda license, ask you whether you accept the default installation location, and whether you want the installation script to intialize conda. 
+Accept all of the defaults and then open and close your terminal for the installation to take effect.
+ 
+#### Installing conda on a Linux machine
+
+Each time you start a new AWS (or many other cloud) machine, you'll have to install miniconda. 
+The installation process is the same as it is on Mac except you'll need to use a different download URL.
+
+```
+curl -JLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh
+``` 
+
+This will prompt you to accept the miniconda license, ask you whether you accept the default installation location, and whether you want the installation script to intialize conda. 
+Accept all of the defaults and then open and close your terminal for the installation to take effect.
+To activate conda, you can either log out and log back in or run:
+
+```
+source ~/.bashrc
+```
+
+### Activating conda and the `(base)` environment
+
+Conda needs to be activated for it to be usable. 
+During the installation process, we configured conda to automatically activate when the shell is started.
+If conda is activated, you should see `(base)` prepend your path. 
+
+The `base` environment contains all of the software needed for conda to run. 
```

Comment:
I'd add a note about how to disable this. I personally don't like it and prefer explicit over implicit. Also a note about what the `base` env means and what's included in it would be useful